doc/guides/writing-and-running-benchmarks.md

```diff
@@ -285,10 +292,11 @@ module, you can use the `--filter` option:*
   --no-progress                 don't show benchmark progress indicator
 ```
 
-For analysing the benchmark results use the `compare.R` tool.
+For analysing the benchmark results, use [node-benchmark-compare][] or the R
+script `benchmark/compare.R`.
 
 ```console
-$ cat compare-pr-5134.csv | Rscript benchmark/compare.R
+$ node-benchmark-compare compare-pr-5134.csv # or cat compare-pr-5134.csv | Rscript benchmark/compare.R
```

Comment:
We can use npx tool to run global dependencies without installing.
```suggestion
$ npx node-benchmark-compare compare-pr-5134.csv # or cat compare-pr-5134.csv | Rscript benchmark/compare.R
```

Reply:
yes, we can but the user was instructed earlier to install the dependency

Reply:
IMHO, this step should be removed to simplify user workflow